src/api/mainContents.ts

```diff
@@ -2,9 +2,13 @@ import { axiosInstance } from '@/api/core';
 import { contents } from '@/types/contents';
 
 export const getMainContents = async (pageParam: number, category: string) => {
-  const response: contents = await axiosInstance.get(
-    `/contents?category=${category}&creator=0&page=${pageParam}&size=10`
-  );
+  const ContentsURL =
+    category === 'all'
+      ? `/contents/all?&page=${pageParam + 1}&size=12&sort=recent`
+      : `/contents?category=${category}&creator=0&page=${
+          pageParam + 1
+        }&size=12&sort=recent`;
```

Comment:
```suggestion
  const pageNum = pageParam + 1;
  const ContentsURL =
    category === 'all'
      ? `/contents/all?&page=${pageNum}&size=12&sort=recent`
      : `/contents?category=${category}&creator=0&page=${pageNum}&size=12&sort=recent`;
```

이렇게 작성하면 보기 더 편할 것 같아용~!